resources/js/data/webauthn/loginWebAuthn.ts

```diff
@@ -48,5 +49,29 @@ export const loginWebAuthn = (Alpine: Alpine) =>
 					})
 					.catch(() => alpine.$dispatch("notify", [{ type: "error", msg: this.error_msg }]));
 			},
+
+			handleKeyUp(event: KeyboardEvent): void {
+				const skipped = ["TEXTAREA", "INPUT", "SELECT"];
+
+				if (document.activeElement !== null && skipped.includes(document.activeElement.nodeName)) {
+					console.log("WAuthn skipped: " + document.activeElement.nodeName);
```

Comment:
A leftover?

Reply:
not really. But I can remove it :)

Reply:
I think it depends what triggers it (and right now I don't even pretend to understand that).

If it's more of an assertion that Just Shouldn't Happen (TM) then I think it's fine to keep it (perhaps after replacing "WAuthn" with "WebAuthn", to be consistent). If on the other hand this message triggers during regular use, then I would remove it -- I don't think we should be printing to the debug log unless something rather unexpected is happening.

Reply:
That small if statement prevents to trigger keybinding behaviors when you are text areas etc...